### DISCUSSION TITLE
i hav an idea

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#455] [(D) Developer]
so like

vpns can modify data right

and some routers can install a vpn

so what if we made a "vpn"

that actually adds a cheat menu on mobile by modifying data

thonk

**Comment by PatheticMustan** [comment]
cool idea, but would be incredibly weird to do some whacky stuff with HTTPS

**Reply by ghost** [reply]
Damn are there any hacks that actually work on mobile :/

**Reply by ArcerionDev** [reply]
no

**Reply by ghost** [reply]
@XxDankGod96xX https://nameisa.github.io/MobileDevice-ProdigyHacking/ i would like to inform you that there's a variety of security reasons relating to why PMGH doesn't want to do this for you officially, so you should make this decision carefully (i.e. if my email was hacked, so would be all the accounts)

**Reply by ghost** [reply]
> cool idea, but would be incredibly weird to do some whacky stuff with HTTPS

wait so actually routers arent affected by HTTPS according to ssd.eff.org thats why the cops can call up your ISP and tell them to hand over your browsing history

**Reply by PatheticMustan** [reply]
ah, cool to know. My biggest concern with such a proxy would be basically leaking all browser history, there's no need for such a thing. I would rather try script based hacks. While yeah, those also have access to the jwt tokens, it's not as bad

**Reply by ghost** [reply]
> ah, cool to know. My biggest concern with such a proxy would be basically leaking all browser history, there's no need for such a thing. I would rather try script based hacks. While yeah, those also have access to the jwt tokens, it's not as bad

it wouldn't leak browsing history unless it intentionally records it

**Reply by PatheticMustan** [reply]
well yeah, but are we really supposed to be given that much trust? It would be trivial to record it and just lie about it

**Reply by ghost** [reply]
oh yeah that is true
